testConnectivity.py

```diff
@@ -220,7 +220,7 @@ def testConnectivity(args):
     if hasattr(args, 'stepSize') is False:
         args.stepSize = 1
     if hasattr(args, 'gemType') is False:
-        args.gemType = None # default to None
+        args.gemType = "ge11" # default to ge11
```

Comment:
I think line 209 might need to be changed to have default `detType` changed to `"short"`?